api/bash/gitline

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+DOMAIN="https://api.github.com"
+source ~/projects/github_bash_project/.privateVariables.sh
+
+# Assigning Arguments
+whichMethod=$1
+username="$2"
+label_state_or_repo_name=$3
+is_private=$4
+
+function fetchIssuesWhereIAmAssignedWithLabel {
+  response=$(curl --silent ${DOMAIN}/search/issues?q=+label:${label_state_or_repo_name}+assignee:${username}+state:open&sort=created&order=asc)
+  echo "$response"
+}
+
+function fetchNotificationsByRepo {
+  response=$(curl -u ${TOKEN}:x-oauth-basic ${DOMAIN}/repos/${username}/${label_state_or_repo_name}/notifications)
```

Comment:
I think instead of `-u $TOKEN` we need to do something like `-H "Authorization: token $TOKEN"` like we have in the example above.